README.md

```diff
@@ -42,7 +42,9 @@ $ ./faros_event.sh CI -k "<faros_api_key>" \
     --run "<cicd_source>://<cicd_organization>/<cicd_pipeline>/<run_uid>" \
     --commit "<vcs_source>://<vcs_organization>/<vcs_repo>/<commit_sha>" \
     --artifact "<artifact_source>://<artifact_org>/<artifact_repo>/<artifact_uid>" \
-    --run_status Success
+    --run_status Success \
+    --run_start_time "1626804346000" \
+    --run_end_time "1626804358000"
```

Comment:
@snabar requested that we include the relevant start and end time fields in our two example events.